### ISSUE TITLE
Survivor - high priority

### ISSUE THREAD

**Issue by GuberPeter** [bertadenes/tippelde#18]
Egy csapatot meg kell nevezni, amelyik a tippelő szerint az adott fordulóban nyer, DE mindegyik csapatot maximum háromszor lehet megnevezni.

Technikailag szerintem egyszerűen legyen egy legördülő lista a forduló meccsei mellett, ahol ki tudnak választani egy csapatot. Nem kell figyelembe venni, hogy melyiket tippelték már korábban, de utána tudjanak nézni ennek valahol.

Kiértékelés szempontjából ugye minden csapat helyes válasz lesz, amelyik abban a körben nyer.

Pontozás:
1. Amíg nem hibázik a tippelő 6-ot, minden helyes tippje után jár 10 pont, utána ilyen bónusz nincs
2. 6 hiba előtt és után is jár 10 pont minden találatért
3. Bónuszpont 8 tippelőnek, akik a legkésőbb érik el a 6 hibát: 1. helyezett 40 pont, majd 35, 30, 25, 20, 15, 10, 5
4. Bónuszpont 8 tippelőnek, akik a legtöbb találatot szerzik a 33 forduló alatt: 1. helyezett 50 pont, majd 45, 40, 35, 30, 25, 20, 15

Ezeket manuálisan írogatom én, ha van rá felület.

**Comment by bertadenes** [comment]
@GuberPeter szamomra az 1. es 2. pont ellent mond, vilagosits fel kerlek.

**Comment by bertadenes** [comment]
@GuberPeter ha fel akalom dolgozni, akkor jo tipp eseten 6 hiba elott 20 jar, 6 hiba utan 10, igaz?

**Comment by bertadenes** [comment]
todo: automatic evaluation after games concluded, automatic due set to games, end of season points

**Comment by GuberPeter** [comment]
Igen, 6 hiba elott 20 pont jar, ha igy nezzuk (csak en ezt eddig kulon tetelkent pontoztam)

**Comment by GuberPeter** [comment]
Azt jelenleg vissza tudják nézni, hogy mikre adtak le Survivor tippeket?

**Comment by GuberPeter** [comment]
Illetve amellett, hogy vissza tudják nézni, még az lenne jó, ha a Games fülön is megjelenne az adott heti Survivor kérdés. Ezt lehet, hogy Tamással megpróbáljuk megcsinálni

**Comment by GuberPeter** [comment]
Most mi a megoldás arra, ha valaki nem tippel egyáltalán? Én tavaly ilyenkor ABC-sorrendben az épp következő elérhető csapatot írtam be, mintha arra tippeltek volna.

**Comment by GuberPeter** [comment]
Az oldalról lezárva a Survivor fordulót (evaluate), az alábbi hibát dobja: 
![image](https://user-images.githubusercontent.com/53101529/62546230-65297680-b863-11e9-800d-0d6ed2932b6f.png)

Adminból sikerül, de pontokat nem látok érte.

**Comment by bertadenes** [comment]
#34 solves games embedding

**Comment by bertadenes** [comment]
@GuberPeter elvileg megcsinaltam a hibat, de le kene tesztelni

**Comment by GuberPeter** [comment]
Elvileg nem jó még
![image](https://user-images.githubusercontent.com/53101529/63043683-04d2ae80-becd-11e9-9015-1dfa13ab6b38.png)

**Comment by bertadenes** [comment]
ott nem is, nem pusholtam herokura amig nincs testelve, itt van fent, le kell huznod es kiprobalni offline
